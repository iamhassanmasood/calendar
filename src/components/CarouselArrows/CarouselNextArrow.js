import React from "react";
import { RightOutlined } from "@ant-design/icons";

const CarouselNextArrow = (props) => {
  const { className, style, onClick } = props;
  return (
    <RightOutlined
      className={`${className} arrows-class`}
      style={{ ...style }}
      onClick={onClick}
    />
  );
};

export default CarouselNextArrow;
