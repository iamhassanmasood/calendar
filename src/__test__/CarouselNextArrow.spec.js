import React from "react";
import { shallow } from "enzyme/build";
import CarouselNextArrow from "./../components/CarouselArrows/CarouselNextArrow";

describe("check CrouselArrpw rendering on dom", () => {
  it("mounts without crashing", () => {
    const wrapper = shallow(<CarouselNextArrow />);
    wrapper.unmount();
  });
  it("check dom tree for snapshots and props", () => {
    let props = {
      className: "",
      style: jest.Obj,
      onClick: jest.fn(),
    };
    let wrapper = shallow(<CarouselNextArrow {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
