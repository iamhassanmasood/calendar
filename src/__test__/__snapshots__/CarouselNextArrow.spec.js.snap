// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check CrouselArrpw rendering on dom check dom tree for snapshots and props 1`] = `ShallowWrapper {}`;
